generate.js

```diff
@@ -26,6 +26,7 @@ function generate (subSectionName, destination, options) {
   allFiles.forEach(file => {
     const template = fs.readFileSync(file, 'utf8')
     const result = template
+      .replace(/%subsection%/g, subSectionName)
```

Comment:
Part of resolving this issue: https://github.com/LaunchPadLab/lp-subsection-generator/issues/19